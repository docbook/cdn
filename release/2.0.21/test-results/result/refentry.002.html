<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for refentry.002.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: refentry.002.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">1</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: refentry.002&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: refentry.002&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="authorgroup"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
                  <span class="tag">&lt;h3&gt;
                     <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Norman&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Walsh&lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="editor"</span>&gt;
                  <span class="tag">&lt;h4 <span class="att">class="editedby"</span>&gt;Edited by&lt;/h4&gt;</span>
                  <span class="tag">&lt;h3&gt;
                     <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Michael (tm)&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Smith&lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/h3&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="othercredit"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Michael (tm)&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Smith&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span <span class="att">class="refentrytitle"</span>&gt;
                        <span class="tag">&lt;a <span class="att">href="#lessminimal"</span>&gt;Also Minimal Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="refpurpose"</span>&gt; - Another test of a minimal RefEntry&lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff del">&lt;article <span class="att">class="refentry"</span> <span class="att">id="lessminimal"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="refnamediv"</span>&gt;
               <span class="tag">&lt;h2&gt;Name&lt;/h2&gt;</span>
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;span <span class="att">class="refname"</span>&gt;Also Minimal&lt;/span&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="refpurpose"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="refpurpose-sep"</span>&gt; — &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="refpurpose-text"</span>&gt;Another test of a minimal RefEntry&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="refpurpose-punc"</span>&gt;.&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;Description&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
               <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                  <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                     <span class="tag">&lt;h4&gt;Subsection&lt;/h4&gt;</span>
                  &lt;/div&gt;</span>
                  <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                  <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                        <span class="tag">&lt;h5&gt;Sub-subsection&lt;/h5&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                        <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                           <span class="tag">&lt;h6&gt;Refsect4-level subsection&lt;/h6&gt;</span>
                        &lt;/div&gt;</span>
                        <span class="tag">&lt;p&gt;Check the indenting.&lt;/p&gt;</span>
                        <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                           <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                              <span class="tag">&lt;h6&gt;Refsect5-level subsection&lt;/h6&gt;</span>
                           &lt;/div&gt;</span>
                           <span class="tag">&lt;p&gt;Check the indenting.&lt;/p&gt;</span>
                        &lt;/div&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/div&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;More Description&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;p&gt;This is a not-so minimal RefEntry.&lt;/p&gt;</span>
               <span class="tag">&lt;div <span class="att">class="formalpara"</span>&gt;
                  <span class="tag">&lt;p&gt;
                     <span class="tag">&lt;b&gt;Ladjwqopnsfil&lt;/b&gt;</span>  This is a formalpara. That is, a paragraph with a title. One
  way to generate output for it is to take the title, “Ladjwqopnsfil”,
  and use it as a “run in” heading that is rendered inline at the
  start of the paragraph, followed by a period (or some other
  punctuation mark).&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;Even More Description&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;p&gt;The following two words, “foo” and “bar”, are both empasized,
separately, with a space between them: <span class="tag">&lt;em&gt;foo&lt;/em&gt;</span>
                  <span class="tag">&lt;em&gt;bar&lt;/em&gt;</span>. Here is the same pair of words in bold, with
a space






 between them: <span class="tag">&lt;strong <span class="att">class="emphasis"</span>&gt;foo&lt;/strong&gt;</span>
                  <span class="tag">&lt;strong <span class="att">class="emphasis"</span>&gt;bar&lt;/strong&gt;</span>
               &lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/article&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: refentry.002&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: refentry.002&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="authorgroup"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
                  <span class="tag">&lt;h3&gt;
                     <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Norman&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Walsh&lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="editor"</span>&gt;
                  <span class="tag">&lt;h4 <span class="att">class="editedby"</span>&gt;Edited by&lt;/h4&gt;</span>
                  <span class="tag">&lt;h3&gt;
                     <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Michael (tm)&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Smith&lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/h3&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="othercredit"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Michael (tm)&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Smith&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span <span class="att">class="refentrytitle"</span>&gt;
                        <span class="tag">&lt;a <span class="att">href="#lessminimal"</span>&gt;Also Minimal Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="refpurpose"</span>&gt; - Another test of a minimal RefEntry&lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;article <span class="att">class="refentry"</span> <span class="att">id="lessminimal"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;div <span class="att">class="refnamediv"</span>&gt;
                     <span class="tag">&lt;h2&gt;Name&lt;/h2&gt;</span>
                     <span class="tag">&lt;p&gt;
                        <span class="tag">&lt;span <span class="att">class="refname"</span>&gt;Also Minimal&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="refpurpose"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="refpurpose-sep"</span>&gt; — &lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="refpurpose-text"</span>&gt;Another test of a minimal RefEntry&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="refpurpose-punc"</span>&gt;.&lt;/span&gt;</span>
                        &lt;/span&gt;</span>
                     &lt;/p&gt;</span>
                  &lt;/div&gt;</span>
                  <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;Description&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                        <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                           <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                              <span class="tag">&lt;h4&gt;Subsection&lt;/h4&gt;</span>
                           &lt;/div&gt;</span>
                           <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                              <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                              <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                                 <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                                    <span class="tag">&lt;h5&gt;Sub-subsection&lt;/h5&gt;</span>
                                 &lt;/div&gt;</span>
                                 <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                                    <span class="tag">&lt;p&gt;This is a minimal RefEntry.&lt;/p&gt;</span>
                                    <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                                       <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                                          <span class="tag">&lt;h6&gt;Refsect4-level subsection&lt;/h6&gt;</span>
                                       &lt;/div&gt;</span>
                                       <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                                          <span class="tag">&lt;p&gt;Check the indenting.&lt;/p&gt;</span>
                                          <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                                             <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                                                <span class="tag">&lt;h6&gt;Refsect5-level subsection&lt;/h6&gt;</span>
                                             &lt;/div&gt;</span>
                                             <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                                                <span class="tag">&lt;p&gt;Check the indenting.&lt;/p&gt;</span>
                                             &lt;/div&gt;</span>
                                          &lt;/div&gt;</span>
                                       &lt;/div&gt;</span>
                                    &lt;/div&gt;</span>
                                 &lt;/div&gt;</span>
                              &lt;/div&gt;</span>
                           &lt;/div&gt;</span>
                        &lt;/div&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/div&gt;</span>
                  <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;More Description&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;This is a not-so minimal RefEntry.&lt;/p&gt;</span>
                        <span class="tag">&lt;div <span class="att">class="formalpara"</span>&gt;
                           <span class="tag">&lt;p&gt;
                              <span class="tag">&lt;b&gt;Ladjwqopnsfil&lt;/b&gt;</span>  This is a formalpara. That is, a paragraph with a title. One
  way to generate output for it is to take the title, “Ladjwqopnsfil”,
  and use it as a “run in” heading that is rendered inline at the
  start of the paragraph, followed by a period (or some other
  punctuation mark).&lt;/p&gt;</span>
                        &lt;/div&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/div&gt;</span>
                  <span class="tag">&lt;div <span class="att">class="refsection"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="refsection-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;Even More Description&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;The following two words, “foo” and “bar”, are both empasized,
separately, with a space between them: <span class="tag">&lt;em&gt;foo&lt;/em&gt;</span>
                           <span class="tag">&lt;em&gt;bar&lt;/em&gt;</span>. Here is the same pair of words in bold, with
a space






 between them: <span class="tag">&lt;strong <span class="att">class="emphasis"</span>&gt;foo&lt;/strong&gt;</span>
                           <span class="tag">&lt;strong <span class="att">class="emphasis"</span>&gt;bar&lt;/strong&gt;</span>
                        &lt;/p&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/div&gt;</span>
               &lt;/div&gt;</span>
            &lt;/article&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: refentry.002</h2><div class="authorgroup"><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><div class="editor"><h4 class="editedby">Edited by</h4><h3><span class="personname"><span class="firstname">Michael (tm)</span> <span class="surname">Smith</span></span></h3></div></div><div class="othercredit"><h3><span class="personname"><span class="firstname">Michael (tm)</span> <span class="surname">Smith</span></span></h3></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span class="refentrytitle"><a href="#lessminimal">Also Minimal Title</a></span><span class="refpurpose"> - Another test of a minimal RefEntry</span></li></ul></div></div><article id="lessminimal" class="refentry"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsection"><div class="refsection-titlepage"><h3>Description</h3></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h4>Subsection</h4></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h5>Sub-subsection</h5></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h6>Refsect4-level subsection</h6></div>
  
  <p>Check the indenting.</p>
  <div class="refsection"><div class="refsection-titlepage"><h6>Refsect5-level subsection</h6></div>
    
    <p>Check the indenting.</p>
  </div>
</div>

</div>
</div>
</div><div class="refsection"><div class="refsection-titlepage"><h3>More Description</h3></div>

<p>This is a not-so minimal RefEntry.</p>
<div class="formalpara"><p><b>Ladjwqopnsfil</b>  This is a formalpara. That is, a paragraph with a title. One
  way to generate output for it is to take the title, “Ladjwqopnsfil”,
  and use it as a “run in” heading that is rendered inline at the
  start of the paragraph, followed by a period (or some other
  punctuation mark).</p></div>
</div><div class="refsection"><div class="refsection-titlepage"><h3>Even More Description</h3></div>

<p>The following two words, “foo” and “bar”, are both empasized,
separately, with a space between them: <em>foo</em>
<em>bar</em>. Here is the same pair of words in bold, with
a space






 between them: <strong class="emphasis">foo</strong> <strong class="emphasis">bar</strong></p>
</div></article></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: refentry.002</h2><div class="authorgroup"><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><div class="editor"><h4 class="editedby">Edited by</h4><h3><span class="personname"><span class="firstname">Michael (tm)</span> <span class="surname">Smith</span></span></h3></div></div><div class="othercredit"><h3><span class="personname"><span class="firstname">Michael (tm)</span> <span class="surname">Smith</span></span></h3></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span class="refentrytitle"><a href="#lessminimal">Also Minimal Title</a></span><span class="refpurpose"> - Another test of a minimal RefEntry</span></li></ul></div></div><div class="content"><article id="lessminimal" class="refentry"><div class="content"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsection"><div class="refsection-titlepage"><h3>Description</h3></div><div class="content">

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h4>Subsection</h4></div><div class="content">

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h5>Sub-subsection</h5></div><div class="content">

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h6>Refsect4-level subsection</h6></div><div class="content">
  
  <p>Check the indenting.</p>
  <div class="refsection"><div class="refsection-titlepage"><h6>Refsect5-level subsection</h6></div><div class="content">
    
    <p>Check the indenting.</p>
  </div></div>
</div></div>

</div></div>
</div></div>
</div></div><div class="refsection"><div class="refsection-titlepage"><h3>More Description</h3></div><div class="content">

<p>This is a not-so minimal RefEntry.</p>
<div class="formalpara"><p><b>Ladjwqopnsfil</b>  This is a formalpara. That is, a paragraph with a title. One
  way to generate output for it is to take the title, “Ladjwqopnsfil”,
  and use it as a “run in” heading that is rendered inline at the
  start of the paragraph, followed by a period (or some other
  punctuation mark).</p></div>
</div></div><div class="refsection"><div class="refsection-titlepage"><h3>Even More Description</h3></div><div class="content">

<p>The following two words, “foo” and “bar”, are both empasized,
separately, with a space between them: <em>foo</em>
<em>bar</em>. Here is the same pair of words in bold, with
a space






 between them: <strong class="emphasis">foo</strong> <strong class="emphasis">bar</strong></p>
</div></div></div></article></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/refentry.002.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>