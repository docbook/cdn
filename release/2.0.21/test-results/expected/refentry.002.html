<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: refentry.002</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: refentry.002</h2><div class="authorgroup"><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><div class="editor"><h4 class="editedby">Edited by</h4><h3><span class="personname"><span class="firstname">Michael (tm)</span> <span class="surname">Smith</span></span></h3></div></div><div class="othercredit"><h3><span class="personname"><span class="firstname">Michael (tm)</span> <span class="surname">Smith</span></span></h3></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span class="refentrytitle"><a href="#lessminimal">Also Minimal Title</a></span><span class="refpurpose"> - Another test of a minimal RefEntry</span></li></ul></div></div><article id="lessminimal" class="refentry"><div class="refnamediv"><h2>Name</h2><p><span class="refname">Also Minimal</span><span class="refpurpose"><span class="refpurpose-sep"> — </span><span class="refpurpose-text">Another test of a minimal RefEntry</span><span class="refpurpose-punc">.</span></span></p></div><div class="refsection"><div class="refsection-titlepage"><h3>Description</h3></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h4>Subsection</h4></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h5>Sub-subsection</h5></div>

<p>This is a minimal RefEntry.</p>

<div class="refsection"><div class="refsection-titlepage"><h6>Refsect4-level subsection</h6></div>
  
  <p>Check the indenting.</p>
  <div class="refsection"><div class="refsection-titlepage"><h6>Refsect5-level subsection</h6></div>
    
    <p>Check the indenting.</p>
  </div>
</div>

</div>
</div>
</div><div class="refsection"><div class="refsection-titlepage"><h3>More Description</h3></div>

<p>This is a not-so minimal RefEntry.</p>
<div class="formalpara"><p><b>Ladjwqopnsfil</b>  This is a formalpara. That is, a paragraph with a title. One
  way to generate output for it is to take the title, “Ladjwqopnsfil”,
  and use it as a “run in” heading that is rendered inline at the
  start of the paragraph, followed by a period (or some other
  punctuation mark).</p></div>
</div><div class="refsection"><div class="refsection-titlepage"><h3>Even More Description</h3></div>

<p>The following two words, “foo” and “bar”, are both empasized,
separately, with a space between them: <em>foo</em>
<em>bar</em>. Here is the same pair of words in bold, with
a space






 between them: <strong class="emphasis">foo</strong> <strong class="emphasis">bar</strong></p>
</div></article></article><script src="../../resources/js/prism.js"></script></body></html>